Declarations/Contracts/Constants.ts

```diff
@@ -18,4 +18,23 @@ export function domainSupportsProperties(domain: Domain): domain is ISupportProp
         domain instanceof PageViewData ||
         domain instanceof RemoteDependencyData ||
         domain instanceof RequestData;
-}
\ No newline at end of file
+}
+
+/**
+ * Subset of Connection String fields which this SDK can parse. Lower-typecased to
+ * allow for case-insensitivity across field names
+ * @type ConnectionStringKey
+ */
+export interface ConnectionString {
+    authorization?: string;
+    instrumentationkey?: string;
```

Comment:
Why not camel case?

Reply:
I guess is because of this line

 const key = kvParts[0].toLowerCase() as ConnectionStringKey;